packages/core/index.d.ts

```diff
@@ -58,11 +58,14 @@ export interface SagaMiddleware<C = {}> extends Middleware {
 
   setContext(props: Partial<C>): void;
 }
+interface onErrorRuntime {
```

Comment:
I'd call it something like `ErrorInfo` or something, `onErrorRuntime` suggests to me that this is a handler function

Reply:
ok, will fix after https://github.com/redux-saga/redux-saga/pull/1703 merged